modules/virt-quick-creating-vm.adoc

```diff
@@ -3,28 +3,33 @@
 // * virt/virtual_machines/virt-create-vms.adoc
 
 :_content-type: PROCEDURE
-[id="virt-quick-creating-vm_{context}"]
-= Quick creating a virtual machine
+[id="virt-creating-vm-template_{context}"]
+= Creating a virtual machine from a template
 
-You can quickly create a virtual machine (VM) by using a template with an available boot source.
+You can create a virtual machine (VM) from a template with an available boot source by using the {product-title} web console.
 
 .Procedure
 
 . Click *Virtualization* -> *Catalog* in the side menu.
 
 . Click *Boot source available* to filter templates with boot sources.
-
 +
 [NOTE]
 ====
-By default, the template list will show only *Default Templates*. Click *All Items* when filtering to see all available templates for your chosen filters.
+By default, the template list displays only *Default Templates*. Click *All Items* when filtering to see all available templates for your chosen filters.
 ====
 
 . Click a template to view its details.
 
-. Click *Quick Create VirtualMachine* to create a VM from the template.
+. Click *Quick create VirtualMachine* to create a VM from the template.
++
+* Optional: Update parameters such as the VM name or disk source:
+
+.. Click *Customize VirtualMachine*.
+.. Edit the parameters and click *Next*.
+.. Click *Create VirtualMachine*.
```

Comment:
These substeps are listed as optional, but that probably doesn't extend to `.. Click *Create VirtualMachine*.` I think it should be a top level step